lte/gateway/python/magma/pipelined/app/classifier.py

```diff
@@ -207,18 +247,19 @@ def add_tunnel_flows(self, precedence:int, i_teid:int,
         actions = []
         if not ue_ip_adr:
```

Comment:
If ue_ip_adr is a required arg, lets fail the function in the beggining. Whenever we have a missing arg we shoudn't be doinf half of the work of the function

Reply:
Yes. Added check beginning of the received grpc message.